Makefile

```diff
@@ -24,7 +24,7 @@ firstup: increase-elk-max-map-count create-docker-sentry-network
 
 .PHONY: adminuser
 adminuser:
-	docker-compose -f docker-compose.yml run --rm app ./manage.py shell -c "from django.contrib.auth.models import User;from django.contrib.auth.models import Group; User.objects.create_superuser('admin', 'crowd@loc.gov', '${CONCORDIA_ADMIN_PW}');Group.objects.create(name='CM')"
+	docker-compose -f docker-compose.yml run --rm app ./manage.py shell -c "from django.contrib.auth.models import User; User.objects.create_superuser('admin', 'crowd@loc.gov', '${CONCORDIA_ADMIN_PW}')"
```

Comment:
This looks functional as long as the password doesn't have shell characters in it. The alternative would be `manage.py createsuperuser --no-input --username=admin --email=crowd@loc.gov` and forcing people to use the password-reset process on the initial install.

Reply:
The main point of this change was to get rid of creating the special "CM" group here (for Community Managers) and put it in a migration. The only reason I haven't changed the process to what you described is because I don't have access to crowd@loc.gov.

Reply:
Okay, works for me.